src/snowflake/snowpark/modin/plugin/_internal/window_utils.py

```diff
@@ -9,7 +9,8 @@
 
 from snowflake.snowpark.modin.plugin.utils.error_message import ErrorMessage
 
-IMPLEMENTED_ROLLING_AGG_FUNCS = ["sum", "mean", "var", "std", "min", "max"]
+IMPLEMENTED_ROLLING_AGG_FUNCS = ["count", "sum", "mean", "var", "std", "min", "max"]
```

Comment:
This two seems exactly the same list, can we just use one here?

Reply:
also those parameter seems only used by test, if that is the case, could you please move it to test folder

Reply:
No we shouldn't use one list b/c there are a bunch of window functions, EWM/Rolling/Expanding/Weighted Window -- keeping them separate b/c we may invest more in implementing some over others. For now, just getting rolling/expanding to parity but going to be focusing more on expanding in future since that has more customer/SE requests.

Reply:
Yeah moving it to the test file makes sense, will do